client/src/components/Sidebar/SidebarHead/SidebarButton/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { SIDEBAR_ITEM } from '../../'
+
+const SidebarButton = ({ value, current, setValue, content }: { value: SIDEBAR_ITEM | null, current: SIDEBAR_ITEM | null, setValue: (arg0: SIDEBAR_ITEM | null) => void, content: any }) => {
```

Comment:
value and current aren't very descriptive variables in my opinion. Probably should add comments and/or make them more descriptive.

Reply:
True, will change the variable name then.